x-pack/docs/en/security/auditing/event-types.asciidoc

```diff
@@ -343,15 +355,21 @@ that have been previously described:
                             uses the <<run-as-privilege, run as authorization functionality>>.
   `authentication.type`::   Method used to authenticate the user.
                             Possible values are `REALM`, `API_KEY`, `TOKEN`, `ANONYMOUS` or `INTERNAL`.
-  `api_key.id`         ::   API key ID returned by the <<security-api-create-api-key,create API key>> request.
+  `apikey.id`          ::   API key ID returned by the <<security-api-create-api-key,create API key>> request.
                             This attribute is only provided for authentication using an API key.
-  `api_key.name`       ::   API key name provided in the <<security-api-create-api-key,create API key>> request.
+  `apikey.name`        ::   API key name provided in the <<security-api-create-api-key,create API key>> request.
                             This attribute is only provided for authentication using an API key.
+  `service_token.name` ::   Name of the <<service-accounts,service account>> token.
+                            This attribute is only provided for authentication using a service account token.
```

Comment:
Maybe mention that the service account namespace and service identifier are audited as the `user.name`.

Reply:
I added a sentence about it under `user.name` where we have a sentence for API key owner as well.